packages/framework/src/Resources/views/Admin/Inline/Menu/menu.html.twig

```diff
@@ -1,8 +1,27 @@
 <ul class="js-toggle-menu navig navig--side">
+
+    <li class="js-toggle-menu-item navig__item">
+        <div class="box-dropdown">
+            <div
+                    class="box-dropdown__select box-dropdown__select--menu js-toggle-menu-trigger"
```

Comment:
js-toggle-menu-trigger seems to be used in templates only, with no related JS

Reply:
removed, thanks!